gnss_ins_sim/pathgen/pathgen.py

```diff
@@ -580,7 +580,7 @@ def gps_gen(ref_gps, gps_err, gps_type=0):
     pos_err = gps_err['stdp'].copy()
     # If position is in the form of LLA, convert gps_err['stdp'] to LLA error
     if gps_type == 0:   # GPS is in the form of LLA, stdp meter to rad
-        earth_param = geoparams.geo_param(ref_gps[0, 1:4])
+        earth_param = geoparams.geo_param(ref_gps[0, 0:3])
```

Comment:
Hi theophilec,

This bug is due to that the very earyly version of the simulation included gps_time as the first column in ref_gps. After making gps_time a separate variable, I forgot to update code there. 
Thanks a lot.

Dong